storage/cloudspanner/log_storage.go

```diff
@@ -52,6 +53,8 @@ const (
 WHERE (t.TreeType = 1 OR t.TreeType = 3)
 AND (t.TreeState = 1 OR t.TreeState = 5)
 AND t.Deleted=false`
+
+	suffixBuckets = 256
```

Comment:
This is used only in one function? I'd inline closer to where it's used, esp. given that the code relies on this being <= 256 (there is a shift left by 8 in one place).

Reply:
done